### PR TITLE
EIP-1559 V2 UI/UX improvements

### PR DESCRIPTION
**Pull request by jpuri**
Small UI / UX fixes in 1559 V2 implementation:

1. hide default settings on advance gas modal for swaps
2. tooltip fixes
3. minification of svg icons added for gas graphs

### CODE REVIEW

File: ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js [hunk 1]
```diff
@@ -90,7 +90,8 @@ const EditGasToolTip = ({
 
   return (
     <div className="edit-gas-tooltip__container">
-      {priorityLevel !== PRIORITY_LEVELS.CUSTOM &&
+      {priorityLevel !== PRIORITY_LEVELS.TEN_PERCENT_INCREASED &&
```
Comment (mcmire) [inline ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js:93]: This condition is a bit hard to parse (I'm surprised that Prettier isn't breaking this up better). What do you think about extracting it to a variable?

**Review by mcmire — DISMISSED** [review]
One little thing, but looks good regardless.

File: ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.test.js [hunk 2]
```diff
@@ -144,4 +128,13 @@ describe('AdvancedGasFeeDefaults', () => {
     fireEvent.click(checkbox);
     expect(mock).toHaveBeenCalledTimes(2);
   });
+
+  it('should not  render anything for swaps', () => {
```
Comment (adonesky1) [inline ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.test.js:132]:
```suggestion
  it('should not render anything for swaps', () => {
```

**Review by adonesky1 — DISMISSED** [review]
Small nit, But otherwise LGTM!

File: ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js (hunk 1 above)
Comment (jpuri) [reply ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js]: good point, I just did that

File: ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.test.js [hunk 3]
```diff
@@ -144,4 +128,13 @@ describe('AdvancedGasFeeDefaults', () => {
     fireEvent.click(checkbox);
     expect(mock).toHaveBeenCalledTimes(2);
   });
+
+  it('should not render option to set default gas options in a swaps transaction', () => {
+    render({}, { editGasMode: EDIT_GAS_MODES.SWAPS });
+    expect(
+      screen.queryByText(
+        'Always use these values and advanced setting as default.',
```
Comment (adonesky1) [inline ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.test.js:136]: should this be "new values" since the option wouldn't have been selected?

Comment (jpuri) [reply ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.test.js]: yep fixed

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js [hunk 4]
```diff
@@ -142,7 +142,7 @@ const CancelSpeedupPopover = () => {
           marginTop={4}
         >
           <Box className="cancel-speedup-popover__edit-gas-button">
-            <EditGasFeeButton />
+            {appIsLoading && <EditGasFeeButton />}
```
Comment (adonesky1) [inline ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js:145]: Not sure I follow what this is achieving?

Comment (jpuri) [reply ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js:145]: EditGasFeeButton displays old estimate,  this change hides the old estimate and wait for it to be updated.

File: ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/priority-fee-input/index.scss [hunk 5]
```diff
@@ -0,0 +1,4 @@
+.advanced-gas-fee-priority-fee-input {
```
Comment (mcmire) [inline ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/priority-fee-input/index.scss:1]: The name of the component is called PriorityFeeInput, should we use that name for consistency?

File: ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js [hunk 6]
```diff
@@ -88,15 +89,20 @@ const EditGasToolTip = ({
     imgAltText = t('curveHighGasEstimate');
   }
 
+  // Gas estimate curve is visible for low/medium/high gas estimates
```
Comment (mcmire) [inline ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js:92]: Perfect, thanks!

File: ui/helpers/utils/gas.js [hunk 7]
```diff
@@ -55,7 +55,7 @@ export function addTenPercentAndRound(hexStringValue) {
   return addTenPercent(hexStringValue, { numberOfDecimals: 0 });
 }
 
-export function isVolatileGasEstimate(estimate) {
+export function isMetamaskSuggestedGasEstimate(estimate) {
```
Comment (mcmire) [inline ui/helpers/utils/gas.js:58]: Nice, makes more sense 👍🏻

**Review by darkwing — DISMISSED** [review]

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js (hunk 4 above)
Comment (adonesky1) [reply ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js:145]: Sorry maybe I'm still confused but shouldn't it be `!appIsLoading`?

File: ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/index.scss [hunk 8]
```diff
@@ -1,15 +1,12 @@
 .edit-gas-tooltip {
```
Comment (digiwand) [inline ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/index.scss:1]: Thanks for cleaning the code! I double-checked to see that this specific class, `.edit-gas-tooltip`, is not declared in the codebase

File: ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js [hunk 9]
```diff
@@ -70,11 +77,11 @@ const AdvancedGasFeeDefaults = () => {
           disabled={hasErrors}
         />
         <Typography variant={TYPOGRAPHY.H7} color={COLORS.UI4} margin={0}>
-          {!isDefaultSettingsSelected && Boolean(advancedGasFeeValues)
-            ? t('advancedGasFeeDefaultOptIn', [
+          {isDefaultSettingsSelected
```
Comment (digiwand) [inline ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js:80]: double-checking, would it make sense to use `isDefaultSettingsSelected || !Boolean(advancedGasFeeValues)` here?

**Review by digiwand — COMMENTED** [review]
Great clean-up! Left a couple of small comments to double-check code

File: ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js (hunk 9 above)
Comment (jpuri) [reply ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js:80]: `isDefaultSettingsSelected` makes UI optimistic and avoid lag resulting from state change.

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js (hunk 4 above)
Comment (jpuri) [reply ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js]: Thanks a lot for bringing this up, I just fixed.

File: app/images/curve-high.svg [hunk 10]
```diff
@@ -1,25 +1 @@
...
...8-.54.3-.133.623-.2.97-.2.373 0 .713.083 1.02.25.307.16.547.367.72.62v-.77h.86v4.71c0 .42-.067.783-.2 1.09-.127.313-.303.57-.53.77a2.17 2.17 0 0 1-.8.46c-.307.107-.64.16-1 .16ZM156.382 15.738v-7.5h.86v3.27c.174-.3.4-.52.68-.66s.584-.21.91-.21c.28 0 .537.047.77.14.234.093.43.23.59.41.167.173.297.387.39.64.094.247.14.523.14.83v3.08h-.85v-2.95c0-.42-.11-.75-.33-.99-.213-.247-.496-.37-.85-.37-.2 0-.39.04-.57.12-.173.08-.326.2-.46.36-.126.153-.23.343-.31.57-.073.227-.11.487-.11.78v2.48h-.86Z" fill="#F66A0A"/><path opacity=".3" d="M45.774 22.738c-10.764 7.42-19.417 8-19.417 8h110.251s-8.653-.58-19.417-8c-10.764-7.42-23.364-22-35.709-22-12.344 0-24.944 14.58-35.708 22Z" fill="#037DD6"/><mask id="b" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="26" y="0" width="111" height="31"><path d="M45.774 22.605c-10.764 7.42-19.417 8-19.417 8h110.251s-8.653-.58-19.417-8c-10.764-7.42-23.364-22-35.709-22-12.344 0-24.944 14.58-35.708 22Z" fill="#EAF6FF"/></mask><g mask="url(#b)"><path fill="#F66A0A" stroke="#fff" stroke-width="2" d="M118.254-5.209h20.706v39.25h-20.706z"/></g></g><defs><clipPath id="a"><path fill="#fff" transform="translate(.813 .082)" d="M0 0h161v31H0z"/></clipPath></defs></svg>
```
[the remark this answers is not in the document]
Comment (jpuri) [reply app/images/curve-high.svg:1]: Good point, but may be not super important as we are not using svg inline.

**Review by adonesky1 — APPROVED** [review]
LGTM

**Review by digiwand — APPROVED** [review]
✨🧹✨

**Review by DELJUJAZI — APPROVED** [review]
